news_items/012_nested_questions/index.html

```diff
@@ -0,0 +1,42 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     width: 70%;
+     margin-left: auto;
+     margin-right: auto;
+ }
+ .code-block > pre {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: 0px;
+     line-height: 125%;
+     overflow: wrap;
+ }
+</style>
+
+<p>
+    <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS BS '19, CS MS '22)
+</p>
+
+<p>
+    Questions now have the ability to be nested inside of subfolders &mdash; meaning that instead of a sprawling <code class="user-output">questions</code> folder inside of your course, you can now organize these into folders however you wish!
+</p>
+
+<img src="nested.png" width="875px">
```

Comment:
I liked your naming scheme where the directories are just `demo` and questions are just `ansiOutput`. To encourage people to use short names, could you change this image to match your desired naming?